media_management_api/settings/base.py

```diff
@@ -256,7 +256,7 @@
 AWS_S3_KEY_PREFIX = SECURE_SETTINGS["aws_s3_key_prefix"]
 
 # CORS headers
-CORS_ORIGIN_ALLOW_ALL = True
+CORS_ALLOW_ALL_ORIGINS = True
```

Comment:
Good catch. Looks like this changed in [django-cors-headers 3.5.0](https://github.com/adamchainz/django-cors-headers/blob/main/HISTORY.rst#350-2020-08-25).